seed/analysis_pipelines/better/pipeline.py

```diff
@@ -139,6 +146,54 @@ def _start_analysis(self):
         ).apply_async()
 
 
+def get_meter_readings(property_id, preprocess_meters):
+    """Returns meters and readings which should meet BETTER's requirements
+
+    :param property_id: int
+    :param preprocess_meters: bool, if true aggregate and interpolate readings
+        into monthly readings. If false, don't do any preprocessing of the property's
+        meters and readings.
+    :return: List[dict], list of dictionaries of the form:
+        { 'meter_type': <Meter.type>, 'readings': List[SimpleMeterReading | MeterReading] }
+    """
+    selected_meters_and_readings = []
+    if preprocess_meters:
+        for meter_type in SEED_TO_BSYNC_RESOURCE_TYPE.keys():
+            meter_readings = MeterReading.objects.filter(meter__property_id=property_id, meter__type=meter_type)
+            if meter_readings.count() == 0:
+                continue
+            monthly_readings = aggregate_meter_readings(meter_readings)
+            monthly_readings = reject_outliers(monthly_readings)
+            # filtering on readings >= 1.0 b/c BETTER flails when readings are less than 1 currently
+            monthly_readings = [reading for reading in monthly_readings if reading.reading >= 1.0]
+            monthly_readings = interpolate_monthly_readings(monthly_readings)
+            if len(monthly_readings) >= 12:
+                selected_meters_and_readings.append({
+                    'meter_type': meter_type,
+                    'readings': monthly_readings
+                })
```

Comment:
Section for preprocessing meters. There are some interesting differences from our original method:
1. we combine readings from the same meter type (instead of keeping different meters separate)
2. we "bin" meters into calendar months
3. we reject outliers
4. we interpolate missing months